04-class-field.tex

```diff
@@ -60,7 +60,7 @@ \section{Property}
 
 \haxe{assets/Property.hx}
 
-For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by
+For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by:
```

Comment:
There you have it, thank you.

On 06-01-2015 08:41, Simon Krajewski wrote:
> In 04-class-field.tex
> <https://github.com/HaxeFoundation/HaxeManual/pull/79#discussion-diff-22517479>:
>
>> @@ -60,7 +60,7 @@ \section{Property}
>>
>>  \haxe{assets/Property.hx}
>>
>> -For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by
>> +For the most part, the syntax is similar to variable syntax, and the same rules indeed apply. Properties are identified by:
>
> Could you remove it? I'll merge afterwards.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HaxeFoundation/HaxeManual/pull/79/files#r22517479>.
>

-- 
Arthur
arthur.szasz@gmail.com
(11) 9 6309 - 0246
(11) 3895 - 6753